### PR TITLE
Add a configuration attribute type for AV1d LST

### PR DESCRIPTION
**Pull request by ZefuLI**
Signed-off-by: zefuli <zefu.li@intel.com>

### CODE REVIEW

File: va/va.h [hunk 1]
```diff
@@ -882,6 +882,13 @@ typedef enum
      *  this setting also could be update by \c VAContextParameterUpdateBuffer
      */
     VAConfigAttribContextPriority       = 41,
+    /** \brief AV1 decoding features.  Read-only.
+     *
+     * This attribute describes the supported features of an
+     * AV1 decoder configuration.  The value returned uses the
+     * VAConfigAttribValDecAV1Features type.
+     */
+    VAConfigAttribValDecAV1Features    = 42,
```
Comment (ChunjuanLiu) [inline va/va.h:891]: The attribute type and returned value type should have different names.
"Val" in the name here should be removed.

File: va/va_dec_av1.h [hunk 2]
```diff
@@ -38,6 +38,26 @@
 extern "C" {
 #endif
 
+/** Attribute value for VAConfigAttribValDecAV1Features.
```
Comment (ChunjuanLiu) [inline va/va_dec_av1.h:41]: VAConfigAttribDecAV1Features -> VAConfigAttribDecAV1Features.

File: va/va_str.c [hunk 3]
```diff
@@ -130,6 +130,7 @@ const char *vaConfigAttribTypeStr(VAConfigAttribType configAttribType)
     TOSTR(VAConfigAttribPredictionDirection);
     TOSTR(VAConfigAttribMultipleFrame);
     TOSTR(VAConfigAttribContextPriority);
+    TOSTR(VAConfigAttribValDecAV1Features);
```
Comment (ChunjuanLiu) [inline va/va_str.c:133]: VAConfigAttribValDecAV1Features -> VAConfigAttribDecAV1Features

File: va/va_dec_av1.h (hunk 2 above)
Comment (ChunjuanLiu) [reply va/va_dec_av1.h:41]: sorry, should be VAConfigAttribValDecAV1Features -> VAConfigAttribDecAV1Features

**Review by ChunjuanLiu — APPROVED** [review]

File: va/va_dec_av1.h [hunk 4]
```diff
@@ -38,6 +38,26 @@
...
+ *
+ */
+typedef union VAConfigAttribValDecAV1Features {
+    struct {
+        /** large scale tile
+         *
+         * This conveys whether AV1 large scale tile is supported by HW.
+         * 0 - unsupported, 1 - supported.
+         */
+        uint32_t lst_support     : 1;
```
[the remark this answers is not in the document]
Comment (ZefuLI) [reply va/va_dec_av1.h]: updated

**Review by ChunjuanLiu — APPROVED** [review]

**Review by XinfengZhang — APPROVED** [review]

**Pull request merged**